### PR TITLE
Desktop: Allow custom CSS when printing #1918 (useCustomCss)

### PR DESCRIPTION
**Pull request by SFoskitt**
https://github.com/laurent22/joplin/issues/1918

@laurent22 Based on the questions added to the issue, here are my responses/test results.

**How does it look when exporting a PDF when no userstyle is present (is the default print style applied?)**

Conditions:
1. `useCustomCss` is default TRUE
2. `userstyle.css` file is absent
3. custom HTML with class names is present

Result:  The PDF export is black and white.  HTML markup is rendered.  No custom styles are applied.

**How does it look when exporting a PDF and a userstyle is present, but not media query (default print style should not be used)**

Conditions:
1. `useCustomCss` is default TRUE
2. `userstyle.css` file is present
3. `userstyle.css` has no media queries
4. custom HTML with class names is present

Result:  The PDF export is rendered with the document styled as the custom CSS.

**How does it look when exporting a PDF and a userstyle is present, and media query too (it should apply these queries to the print style)**

Conditions:
1. `useCustomCss` is default TRUE
2. `userstyle.css` file is present
3. `userstyle.css` contains media queries for max-width 480px
4. `userstyle.css` contains media queries for min-width 481px and max-width 750px
5. custom HTML with class names is present

Result:  Media query styles work as expected for screen rendering, PDF export, and printing.  (Mac Mojave)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ Joplin uses and renders a Github-flavoured Markdown with a few variations and ad
...
+Rendered markdown can be customized by placing a userstyle file in the profile directory `~/.config/joplin-desktop/userstyle.css` (This path might be different on your device - check at the top of the Config screen for the exact path). This file supports standard CSS syntax. Joplin ***must*** be restarted for the new css to be applied, please ensure that Joplin is not closing to the tray, but is actually exiting. Note that this file is used for both displaying the notes and printing the notes.  Be aware how the CSS may look printed (for example, printing white text over a black background is usually not wanted).
```
Comment (tessus) [inline README.md]: `and printing the notes.  Be aware how the CSS ` There are 2 spaces between `notes.` and `Be`.

**Comment by tessus** [comment]
Other than that it looks good and thank you for working on this.

File: README.md (hunk 1 above)
Comment (SFoskitt) [reply README.md]: This is corrected.

**Comment by SFoskitt** [comment]
README.md is updated and the changes to `package.json` and `package-lock.json` are reversed.  @tessus, please let me know if there is anything else I can do?  Thank you!

**Comment by laurent22** [comment]
Perfect, thanks for the pull request @SFoskitt!

**Pull request merged**